src/ncp/ncp_base_radio.cpp

```diff
@@ -459,6 +480,42 @@ template <> otError NcpBase::HandlePropertySet<SPINEL_PROP_RCP_MAC_KEY>(void)
     return error;
 }
 
+template <> otError NcpBase::HandlePropertyGet<SPINEL_PROP_RCP_MAC_FRAME_COUNTER>(void)
+{
+    return mEncoder.WriteUint32(otLinkRawGetMacFrameCounter(mInstance));
+}
+
+template <> otError NcpBase::HandlePropertyGet<SPINEL_PROP_RCP_STORED_MAC_FRAME_COUNTER>(void)
+{
+    return mEncoder.WriteUint32(otLinkRawGetStoredMacFrameCounter(mInstance));
+}
+
+template <> otError NcpBase::HandlePropertySet<SPINEL_PROP_RCP_MAC_FRAME_COUNTER>(void)
+{
+    otError  error = OT_ERROR_NONE;
+    uint32_t frameCounter;
+
+    SuccessOrExit(error = mDecoder.ReadUint32(frameCounter));
+
+    error = otLinkRawSetMacFrameCounter(mInstance, frameCounter, &NcpBase::LinkRawMacFrameCounterStore);
```

Comment:
Can we set the store callback at initialization instead of passing it every time we want to set and store the frame counter?